_includes/help/nav_sidenav.html

```diff
@@ -5,14 +5,13 @@
       {% for subpage in site.help_pages %}
       {% assign subpage_slug = subpage | slugify %}
       {% assign related_questions = site.help | where_exp: "item", "item.url contains subpage_slug" | sort: 'order' %}
-      {% capture is_initially_expanded %}
-      {% if page.url contains subpage_slug %}true{% else %}false
+      {% if page.url contains subpage_slug %}
+        {% assign is_initially_expanded = 'true' %}
+        {% assign is_initially_hidden = 'false' %}
+      {% else %}
+        {% assign is_initially_expanded = 'false' %}
+        {% assign is_initially_hidden = 'true' %}
       {% endif %}
-      {% endcapture %}
-      {% capture is_initially_hidden %}
-      {% if page.url contains subpage_slug %}false{% else %}true
-      {% endif %}
-      {% endcapture %}
```

Comment:
Another option would be some `{%-`'s to remove whitespace?

I tried this and it appears to work:

```
      {%- capture is_initially_expanded -%}
        {% if page.url contains subpage_slug %}true{% else %}false{% endif %}
      {%- endcapture -%}
      {%- capture is_initially_hidden -%}
        {% if page.url contains subpage_slug %}false{% else %}true{% endif %}
      {%- endcapture -%}
```

**EDIT** nevermind, I think the nested inverted `if`s are much harder to read than the assignments

Reply:
> Another option would be some `{%-`'s to remove whitespace?

While I agree with your edit, I also admit I wasn't aware of this whitespace control feature. TIL!